magpy/gui/magpy_gui.py

```diff
@@ -2086,6 +2090,21 @@ def onOpenAuxButton(self, event):
     def changeStatusbar(self,msg):
         self.SetStatusText(msg)
 
+    def UpdateCursorStatus(self, event):
+        """Motion event for displaying values under cursor."""
+        if event.inaxes:
```

Comment:
Since the else for this if is so short, it would be simpler to handle that case first and return, e.g.:

```
if not event.inaxes:
    self.changeStatusbar("Ready")
    return
# do stuff when event.inaxes is true
```